lethal-squirrels/workforce-management
# Added Training Program View Functionality

## REFERENCE

Ticket: (#9)

## DESCRIPTION
Added a statement to check if the training program is in the future, then display only those future programs in the Training Program index view.
Removed the Edit and Delete Options as they are not mentioned **(unsure about this, I can add them back if we feel it's a better option)

The Create Training Program View did not need changes.

## STEPS TO TEST
1. Clone this branch to a local repo
2. Look at the code. 
3. Test functionality
	* Ensure the Program Runs
	* go to `/trainingprograms/`
	* Ensure only the training programs shown are in the future
        * Only a Details option should be shown next to the programs
